driver.py

```diff
@@ -602,7 +618,7 @@ def write_run_config(path, **kwargs):
         "name": name,
         "path": path,
         "smartsim_version": smartsim.__version__,
-        "smartredis_version": "0.3.0", # TODO put in smartredis __version__
+        "smartredis_version": "0.4.0", # TODO put in smartredis __version__
```

Comment:
this should be ``smartredis.__version__`` but we still need to put that into the python client library in the top level __init__.py. Either way, the next version of SmartRedis will be 0.3.1 if thats what you were going for here.

Reply:
I'll tweak this to 0.3.1 then as a stopgap measure